Factorial/Factorial/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Factorial
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine($"5! = {Factorial(5)}");
+        }
+
+        public static int Factorial(int n)
+        {
+            if (n <= 0)
+                return 1;
```

Comment:
i suggest to have space after this line, because `return n * Factorial(n - 1);` another block of operations. We have to separate by new lines different logical block to increase readability

Reply:
Ok, will follow this rule))